src/sentry/tasks/integrations.py

```diff
@@ -19,37 +19,85 @@
 logger = logging.getLogger('sentry.tasks.integrations')
 
 
+def should_comment_sync(installation, external_issue):
+    organization = Organization.objects.get(id=external_issue.organization_id)
+    has_issue_sync = features.has('organizations:integrations-issue-sync',
+                                  organization)
+    return has_issue_sync and installation.should_sync('comment'), installation
+
+
 @instrumented_task(
-    name='sentry.tasks.integrations.post_comment',
+    name='sentry.tasks.integrations.create_comment',
     queue='integrations',
     default_retry_delay=60 * 5,
     max_retries=5
 )
 # TODO(jess): Add more retry exclusions once ApiClients have better error handling
 @retry(exclude=(ExternalIssue.DoesNotExist, Integration.DoesNotExist))
-def post_comment(external_issue_id, data, user_id, **kwargs):
```

Comment:
You might want to leave a no-op `post_comment` or an alias to `create_comment` around. You can't be certain that when this is deployed that there won't be any pending jobs in the queues.